packages/typespec-vscode/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "typespec-vscode",
-  "version": "0.64.0",
+  "version": "0.64.1",
```

Comment:
this should be done as the next pr, here the changelogs are not converted to the changelog.md.

If you do it in this PR the CI will fail for missing changelog. YOu have to name the branch `publish/xyz` for it to ignore them